src/routes/[slug]/+page.svelte

```diff
@@ -27,10 +27,10 @@
 	let lightboxIndex = $state(0);
 	let showLightbox = $state(false);
 
-	const url = new URL(window.location.href);
-	const twitterShareURL = `https://twitter.com/intent/tweet?text=${url.toString()}`;
-	const facebookShareURL = `https://www.facebook.com/sharer/sharer.php?u=${url.toString()}`;
-	const linkedinShareURL = `https://www.linkedin.com/shareArticle?mini=true&url=${url.toString()}`;
+	// NEW - Initialize as state variables instead
+	let twitterShareURL = $state('');
+	let facebookShareURL = $state('');
+	let linkedinShareURL = $state('');
```

Comment:
In this case, if the problem is that window isn't available at the time of the variables initialization, you can use the page store, like the example below.

```suggestion
        import { page } from '$app/stores'

	const twitterShareURL = `https://twitter.com/intent/tweet?text=${$page.url.href}`
```